packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/aspnet/remotesignout.md

```diff
@@ -0,0 +1,26 @@
+Open the controller where you will handle the Okta sign out process and add an `OktaSignOut` method. This time you need to also sign the user out of the OIDC middleware:
```

Comment:
Open the controller where you handle the Okta sign-out process and add an `OktaSignOut` method. This time you need to also sign the user out of the OIDC middleware:

Reply:
Same comment here - have developers modify the sign out method they already created.